packages/core/src/StarknetWindowObject.ts

```diff
@@ -1,12 +1,69 @@
-import type { AccountInterface, ProviderInterface } from "starknet"
-import type {
-  AccountInterface as AccountInterfaceV4,
-  ProviderInterface as ProviderInterfaceV4,
-} from "starknet4"
+export enum StarknetChainId {
+  SN_MAIN = "0x534e5f4d41494e",
+  SN_GOERLI = "0x534e5f474f45524c49",
+}
+
+export enum Permission {
+  Accounts = "accounts",
+}
+
+type FELT = string
 
-export type AccountChangeEventHandler = (accounts: string[]) => void
+type Call = {
```

Comment:
Call type : This type name already exists in Starknet.js, but their definition are not the same, generating confusion when coding. My proposal is to rename in the interface `Call` to `CallParameters`.